test/e2e/scheduling/priorities.go

```diff
@@ -250,11 +287,36 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		})
 		framework.ExpectNoError(f.WaitForPodRunning(pod.Name))
 
-		By("Pod should prefer scheduled to the node that pod can tolerate.")
+		By("Pod should prefer to be scheduled to the node that pod can tolerate.")
 		tolePod, err = cs.CoreV1().Pods(ns).Get(tolerationPodName, metav1.GetOptions{})
 		Expect(err).NotTo(HaveOccurred())
 		Expect(tolePod.Spec.NodeName).To(Equal(nodeName))
 	})
+
+	It("Pod should prefer to be scheduled to nodes having container images ready.", func() {
+		// decide the images used by the pod and the node that has the image
+		nodeWithUsedImage := nodeList.Items[0]
+		// we use the mnist-tpu as the test image used by the pod, which has relatively large image size
+		usedImages := []imageutils.ImageConfig{
+			imageutils.MnistTpu,
+		}
+		// make the node have the given images
+		framework.ExpectNoError(createImagePodForNode(f, cs, ns, nodeWithUsedImage, usedImages))
```

Comment:
Will other priorities feature impact the result?

Reply:
The size of `gcr.io/kubernetes-e2e-test-images/mnist-tpu:1.0` used in this test will ensure `MaxPriority` of `ImageLocalityPriority`. I assume this as a guarantee.

Reply:
The image size of `gcr.io/kubernetes-e2e-test-images/mnist-tpu` is ~430mb, and the `MaxPriority` requires 1000 mb+ image.

Reply:
gcr.io/kubernetes-e2e-test-images/mnist-tpu   1.0                 481eb895fc2d        3 months ago        1.49GB

Reply:
I wonder if you were testing with cri-containerd which reports compressed size?  This priority need to use virtual size in 1.11 to align with e2e tests which use docker. i.e. The threshold of 1000mb should be updated when e2e test migrate to cri-containerd. 

Please note this does not affect user's experience regardless of what runtime he is using.

Reply:
@k82cn oops, yes, for the integration test I should report the 1.49GB on disk ("real") size @resouer reports too. Addressed it. Thanks for catching it!

"Virtual size" is a bit ambiguous - I suppose "decompressed size" is what being referred to here?

Reply:
Using a larger image to cover both cases is better, but after all this does not affect the priority algorithm itself. So I am fine with ethier size as long as it works with current e2e test.

But if u chose to use virtual size please make sure it’s consistent in test code and leave a todo to clarify.

Reply:
Got it - It is consistent and with TODO comment mentioning the LargeImage we should use in the future.

Reply:
@k82cn Yes, it is virtual size we are using in this priority test because it's what Docker reports in current node status.